conf.d/splunk_topology.yaml.example

```diff
@@ -0,0 +1,31 @@
+# Splunk topology intake uses basic authentication
+
+init_config:
+  default_timeout: 10
+  max_retry_count: 5
+  seconds_between_retries: 1
+
+  # Polling interval between calls to splunk
```

Comment:
Its not an interval between calls to splunk, but instead between dispatching all saved searches. So something like "Interval at which to dispatch all saved search." Maybe change the name to reflect this as well to include the time unit.